frame/staking/src/lib.rs

```diff
@@ -679,6 +680,21 @@ impl Default for Forcing {
 	fn default() -> Self { Forcing::NotForcing }
 }
 
+// A value placed in storage that represents the current version of the Balances storage.
+// This value is used by the `on_runtime_upgrade` logic to determine whether we run
+// storage migration logic. This should match directly with the semantic versions of the Rust crate.
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+enum Releases {
```

Comment:
Seeing this duplicated struct everywhere and with the problem of each module naming it whatever it wants, and as we had here confused docs, maybe it is a good idea to make it 

```
// support
trait RuntimeStorageVersion<V>: Parameter {
  fn get_version() -> V
}
```

and each module can implement whatever it wants for this

Reply:
Let's keep this in mind, and see how much this is used and if it is really needed.